app/templates/events/event_list.html

```diff
@@ -1,8 +1,106 @@
-<h1>Event List</h1>
-
-<ul>
-    {{user}}
-    {% for event in events %}
-        <li>{{event}} </li>
-    {% endfor %}
-</ul>
+{% extends "base.html"%}
+{% block scripts %}
+  {{super()}}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.1/js/bootstrap-select.js"></script>
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+  </script>
+{% endblock %}
+{% block styles %}
+  {{super()}}
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.12.1/css/bootstrap-select.css" />
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+{% endblock %}
+{% block app_content %}
+
+
+<h1 class="text-center">Event List for {{termName}}</h1>
+<div>
+  <ul class="nav nav-pills nav-fill mb-3" id="pills-tab" role="tablist">
+  <li class="nav-item" role="presentation">
+    <button class="nav-link active" id="studentLedEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-student-led" type="button" role="tab" aria-controls="pills-student-led" aria-selected="true">Student Led Service</button>
+  </li>
+  <li class="nav-item" role="presentation">
+    <button class="nav-link" id="trainingEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-training" type="button" role="tab" aria-controls="pills-training" aria-selected="false">Training and Education</button>
+  </li>
+  <li class="nav-item" role="presentation">
+    <button class="nav-link" id="bonnerScholarsEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-bonner-scholars" type="button" role="tab" aria-controls="pills-bonner-scholars" aria-selected="false">Bonner Scholars</button>
+  </li>
+  <li class="nav-item" role="presentation">
+    <button class="nav-link" id="oneTimeEvents"
+    data-bs-toggle="pill" data-bs-target="#pills-one-time" type="button" role="tab" aria-controls="pills-one-time" aria-selected="false">One-time Events</button>
+  </li>
+  </ul>
+</div>
+{% macro createAccordion(categoryPrograms) %}
+  <div class="accordion" id="categoryAccordion">
+    {% for program,events in categoryPrograms.items() %}
+      <div class="accordion-item">
+        <h2 class="accordion-header" id="accordion__header_{{program}}">
+          <button class="accordion-button collapsed"
+                  type="button"
+                  data-bs-toggle="collapse"
+                  data-bs-target="#accordion__body_{{program}}_{{ loop.index }}"

```

Comment:
loop.index could have a conflict with another tab. For example, if Adopt-A-Grandparent is always first in the list, it will have the same ID each time.

Reply:
added a parameter to the createAccordion macro mainly so that it could be used in the id. passed in differnt integer for that parameter when we use the createAccordion macro for different pill.